runtime/container.go

```diff
@@ -193,6 +193,17 @@ func Load(root, id, shimName string, timeout time.Duration) (Container, error) {
 		}
 		c.processes[pid] = p
 	}
+
+	_, err = os.Stat(c.bundle)
+	if err != nil && !os.IsExist(err) {
+		for key, p := range c.processes {
+			if key == "init" {
```

Comment:
nit: `runtime.InitProcessID` (I know the current code is bad as using it everywhere though)